app/controllers/contacts/overview/persons.js

```diff
@@ -0,0 +1,82 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class ContactsOverviewPersonsController extends Controller {
+  @tracked selectedContact;
+  @tracked showContactItemModal = false;
+
+  @tracked sort = '-full-name';
+  @tracked page = 0;
+  @tracked size = 25;
+
+  @action
+  prevPage() {
+    if (this.page > 0) {
+       this.page -= 1;
+    }
+  }
+
+  @action
+  nextPage() {
+    this.page += 1;
+  }
+
+  @action
+  setPageSize(size) {
+    this.size = size;
+    this.page = 0;
+  }
+
+  @action
+  setSort(sort) {
+    this.sort = sort;
+  }
+
+  @action
+  openContactItem(contact) {
+    this.selectedContact = contact;
+    this.showContactItemModal = true;
+  }
+
+  @action
+  async deleteContact() {
+    await this.selectedContact.destroyRecord();
+    this.send('reloadModel');
+    this.closeContactItemModal();
+  }
+
+  @action
+  async changeContact() {
+    const telephone = await this.selectedContact.telephone;
```

Comment:
Volgens wat in de story staat zou een contact item 2 telefoonnummers moeten hebben. Werd er beslist dit te beperken tot 1 telefoonnummer of werd dit over het hoofd gezien?

Reply:
dit is overal zo gedaan geweest, ik heb dat gedaan aan de hand van de modellen die in de FR stonden.